airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/DebeziumRecordPublisher.java

```diff
@@ -86,7 +86,15 @@ public void start(Queue<ChangeEvent<String, String>> queue) {
           // more on the tombstone:
           // https://debezium.io/documentation/reference/configuration/event-flattening.html
           if (e.value() != null) {
-            queue.add(e);
+            boolean inserted = false;
+            while (!inserted) {
+              inserted = queue.offer(e);
+              try {
+                Thread.sleep(10);
```

Comment:
btw this should only trigger if `inserted == false`

Reply:
Ah good catch! sorry about that!